### ISSUE TITLE
Flight Log.csv not updated

### ISSUE THREAD

**Issue by RaphaelFaria123** [Marginal/EDMarketConnector#142]
Was this feature removed? I have been developing a mobile app/website which parses the Flight Log.csv to infer a list of trades. The trades, along with data from eddb.io, are saved to mongo so that a trade history can be viewed on mobile or the web.
  
You can view my web mockups here. [Adobe XD prototype](https://xd.adobe.com/view/ec2fcede-9909-452b-4cb4-609aba946514/)

I've attached screen caps from functional Ionic app.

I see there is a way to write custom plugins, which I'm thinking of using to send market data to my server which parses the Flight Log.csv files and uses the market prices to calculate profits.

I could also develop my own flight log.csv exporter, but seeing that the Flight Log.csv file export was available in older version.... I was hoping that feature could be reintroduced?

Also, I've noticed that the keyboard shortcut doesn't work if the automatic update on docking flag is off. 
<img width="376" alt="screen shot 2016-09-07 at 5 33 06 am" src="https://cloud.githubusercontent.com/assets/4953640/18298938/da3ac6a6-74bc-11e6-8659-3b97e8613f8d.png">
<img width="380" alt="screen shot 2016-09-07 at 5 31 40 am" src="https://cloud.githubusercontent.com/assets/4953640/18298943/e56b5720-74bc-11e6-987a-a7d51b2aa448.png">

**Comment by Marginal** [comment]
Looks nice.

The flight log feature was dropped in 2.15. Please use [this](https://github.com/Marginal/EDMCFlightLog) EDMC plugin which should provide the same functionality.

> I've noticed that the keyboard shortcut doesn't work if the automatic update on docking flag is off. 

I can't reproduce this. Please check whether you have the "Only when Elite: Dangerous is the active app" setting selected, and that you're not in the "cooldown" period.

**Comment by RaphaelFaria123** [comment]
Thank you very much!

I've added the plugin and it's working great - I'm up and running again. The code will also help me kick start my development for the EDMC plugin I need to write.

About the problem I was having with the keyboard shortcut...

I also am no longer able to reproduce the issue. Maybe my MacBook pro was in some kind of state  where it thought that ED was not the active app.  

Automatic update was most probably just a red herring. I think I did exit the game when I changed that setting and fired it up again...
